docs/fundamentals/flow/health-check.md

```diff
@@ -331,3 +331,46 @@ And you will get a valid empty response indicating the Gateway's ability to serv
 ```json
 {}
 ```
+
+## Use jina ping to do health checks
+
+Once a Flow is running, you can use `jina ping` CLI  {ref}`CLI <../api/jina_cli>` to run readiness check of the complete Flow or of individual Executors or Gateway.
+
+Let's start a Flow in the terminal by executing the following python code:
+
+```python
+from jina import Flow
+
+with Flow(protocol='grpc', port=12345).add(port=12346) as f:
+    f.block()
+```
+
+We can check the readiness of the Flow:
+
+```bash
+jina ping flow grpc://localhost:12345
+```
+
+Also we can check the readiness of an Executor:
+
+```bash
+jina ping executor localhost:12346
```

Comment:
Do we support the same syntax of host for flow, gateway and executor ? Imo it should be [optional scheme://]hostname:port and default to grpc. I also think it's nice to have these tested in cli tests

Reply:
For Executor it does not make sense, because it is always `grpc`.

Reply:
I added the code not to consider `scheme` if it comes along